### ISSUE TITLE
How to handle Titlings with multiple children

### ISSUE THREAD

**Issue by greenTara** [ag-csw/OntoMaven#23]
Lines 61+ in Includes:

if (children.size() == 1) {
newRootElement = children.get(0);
newRootElement.detach();
} else {
Element constructElement = new Element("Construct", XMLUtil.NS_XCL2);
for (Element child : children) {
child.detach();
constructElement.addContent(child);
}
newRootElement = constructElement;
}

That's pretty sneaky - at some point we'll want to deconstruct that.

What we really should be doing is locating the (one) child that is a CL2 text (an Import, a Construct or a Restrict) and use that as the new root.
